src/test/e2e/00_use_cli_test.go

```diff
@@ -16,126 +16,174 @@ import (
-	require.Error(t, err)
-
-	// Test that changing the log level actually applies the requested level
-	_, stdErr, _ = e2e.ExecZarfCommand("version", "--log-level=debug")
-	expectedOutString := "Log level set to debug"
-	require.Contains(t, stdErr, expectedOutString, "The log level should be changed to 'debug'")
-
-	// Test that `zarf package deploy` gives an error if deploying a remote package without the --insecure or --shasum flags
-	stdOut, stdErr, err = e2e.ExecZarfCommand("package", "deploy", "https://zarf-examples.s3.amazonaws.com/zarf-package-appliance-demo-doom-20210125.tar.zst", "--confirm")
-	require.Error(t, err, stdOut, stdErr)
-
-	pkgName := fmt.Sprintf("zarf-package-dos-games-%s.tar.zst", e2e.Arch)
-
-	_ = os.Mkdir(otherTmpPath, 0750)
-	stdOut, stdErr, err = e2e.ExecZarfCommand("package", "create", "examples/dos-games", "--confirm", "--zarf-cache", cachePath, "--tmpdir", otherTmpPath, "--log-level=debug")
-	require.Contains(t, stdErr, otherTmpPath, "The other tmp path should show as being created")
-	require.NoError(t, err, stdOut, stdErr)
-
-	stdOut, stdErr, err = e2e.ExecZarfCommand("package", "inspect", pkgName, "--tmpdir", otherTmpPath, "--log-level=debug")
-	require.Contains(t, stdErr, otherTmpPath, "The other tmp path should show as being created")
-	require.NoError(t, err, stdOut, stdErr)
-
-	e2e.CleanFiles(pkgName)
-
-	files, err := os.ReadDir(imageCachePath)
-	require.NoError(t, err, "Encountered an unexpected error when reading image cache path")
-	require.Greater(t, len(files), 1)
-
-	// Test removal of cache
-	stdOut, stdErr, err = e2e.ExecZarfCommand("tools", "clear-cache", "--zarf-cache", cachePath)
-	require.NoError(t, err, stdOut, stdErr)
-
-	// Check that ReadDir returns no such file or directory for the cachePath
-	_, err = os.ReadDir(cachePath)
-	if runtime.GOOS == "windows" {
-		msg := fmt.Sprintf("open %s: The system cannot find the file specified.", cachePath)
-		require.EqualError(t, err, msg, "Did not receive expected error when reading a directory that should not exist")
-	} else {
-		msg := fmt.Sprintf("open %s: no such file or directory", cachePath)
-		require.EqualError(t, err, msg, "Did not receive expected error when reading a directory that should not exist")
-	}
-
-	// Test generation of PKI
-	tlsCA := "tls.ca"
-	tlsCert := "tls.crt"
-	tlsKey := "tls.key"
-	stdOut, stdErr, err = e2e.ExecZarfCommand("tools", "gen-pki", "github.com", "--sub-alt-name", "google.com")
-	require.NoError(t, err, stdOut, stdErr)
-	require.Contains(t, stdErr, "Successfully created a chain of trust for github.com")
-
-	_, err = os.ReadFile(tlsCA)
-	require.NoError(t, err)
-
-	_, err = os.ReadFile(tlsCert)
-	require.NoError(t, err)
-
-	_, err = os.ReadFile(tlsKey)
-	require.NoError(t, err)
-
-	e2e.CleanFiles(shasumTestFilePath, cachePath, otherTmpPath, pkgName, tlsCA, tlsCert, tlsKey)
+
+	t.Run("zarf prepare sha256sum <local>", func(t *testing.T) {
```

Comment:
I would prefer having the functions be given their own named functions instead of being written inline.

I don't think I'd block the PR over it, but it would make me happier <3